scripts/interchain_security_testnet.sh

```diff
@@ -0,0 +1,235 @@
+
+mv ${CONS_NODE_DIR}/edited_genesis.json ${CONS_NODE_DIR}/config/genesis.json &&
+    rm ccvconsumer_genesis.json
+
+echo '{"height": "0","round": 0,"step": 0}' > ${CONS_NODE_DIR}/data/priv_validator_state.json  
+cp ${PROV_NODE_DIR}/config/priv_validator_key.json ${CONS_NODE_DIR}/config/priv_validator_key.json  
+cp ${PROV_NODE_DIR}/config/node_key.json ${CONS_NODE_DIR}/config/node_key.json
+
+sed -i -r "/node =/ s/= .*/= \"tcp:\/\/${MY_IP}:26648\"/" ${CONS_NODE_DIR}/config/client.toml
+
+# consumer local node use the following command
+interchain-security-cd start --home $CONS_NODE_DIR \
+        --rpc.laddr tcp://${MY_IP}:26648 \
+        --grpc.address ${MY_IP}:9081 \
+        --address tcp://${MY_IP}:26645 \
+        --p2p.laddr tcp://${MY_IP}:26646 \
+        --grpc-web.enable=false \
+        &> ${CONS_NODE_DIR}/logs &
+
+tail -f ${CONS_NODE_DIR}/logs
+
+tee ~/.hermes/config.toml<<EOF
+[global]
+ log_level = "info"
+
+[[chains]]
+account_prefix = "cosmos"
+clock_drift = "5s"
+gas_multiplier = 1.1
+grpc_addr = "tcp://${MY_IP}:9081"
+id = "$CONS_CHAIN_ID"
+key_name = "relayer"
+max_gas = 2000000
+rpc_addr = "http://${MY_IP}:26648"
+rpc_timeout = "10s"
+store_prefix = "ibc"
+trusting_period = "14days"
+websocket_addr = "ws://${MY_IP}:26648/websocket"
+
+[chains.gas_price]
+       denom = "orai"
+       price = 0.00
+
+[chains.trust_threshold]
+       denominator = "3"
+       numerator = "1"
+
+[[chains]]
+account_prefix = "orai"
+clock_drift = "5s"
+gas_multiplier = 1.1
+grpc_addr = "tcp://${MY_IP}:9091"
+id = "$PROV_CHAIN_ID"
+key_name = "relayer"
+max_gas = 2000000
+rpc_addr = "http://${MY_IP}:26658"
+rpc_timeout = "10s"
+store_prefix = "ibc"
+trusting_period = "14days"
+websocket_addr = "ws://${MY_IP}:26658/websocket"
+
+[chains.gas_price]
+       denom = "orai"
+       price = 0.00
+
+[chains.trust_threshold]
+       denominator = "3"
+       numerator = "1"
+EOF
+
+#Delete all previous keys in relayer
+hermes keys delete --chain consumer --all
+hermes keys delete --chain provider --all
+
+#Import accounts key
+hermes keys add --key-file  ${CONS_NODE_DIR}/${CONS_KEY}.json --chain consumer
+hermes keys add --key-file  ${PROV_NODE_DIR}/${PROV_KEY}.json --chain provider
+
+hermes create connection \
+     --a-chain consumer \
+    --a-client 07-tendermint-0 \
+    --b-client 07-tendermint-0
+
+hermes create channel \
+    --a-chain consumer \
+    --a-port consumer \
+    --b-port provider \
+    --order ordered \
+    --channel-version 1 \
+    --a-connection connection-0
+
+pkill -f hermes  
+hermes --json start &> ~/.hermes/logs &
+
+tail -f ~/.hermes/logs
+
+
+# Get validator delegations
+DELEGATIONS=$(oraid q staking delegations \
+  $(jq -r .address ${PROV_NODE_DIR}/${PROV_KEY}.json) --home $PROV_NODE_DIR -o json)
+
+# Get validator operator address
+OPERATOR_ADDR=$(echo $DELEGATIONS | jq -r '.delegation_responses[0].delegation.validator_address')
+
+# Delegate tokens
+oraid tx staking delegate $OPERATOR_ADDR 6900000orai \
```

Comment:
Will this transaction be catched by the relayer and relay it to the consumer chain?

Reply:
Exactly!